src/tasks/leveling.ts

```diff
@@ -938,7 +1066,7 @@ export const LevelingQuest: Quest = {
     {
       name: "Monster Habitats",
       ready: () =>
-        get("monsterHabitatsFightsLeft") > 0 &&
+        toInt(get("monsterHabitatsFightsLeft")) > 0 &&
```

Comment:
Why is this necessary? Libram should already be casting it to the right datatype